### PR TITLE
WIP: Object detection tutorial

### PR DESCRIPTION
**Pull request by briacht**
## Summary

Add tutorial for Object Detection in Model Builder

[Preview Link](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-model-builder?branch=pr-en-us-21192)

### CODE REVIEW

File: docs/machine-learning/tutorials/object-detection-model-builder.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+1. [Download the dataset](https://datasetlinkhere) of 50 stop sign images and unzip.
+1. [Download VoTT](https://github.com/Microsoft/VoTT/releases) (Visual Object Tagging Tool).
+1. Open VoTT and select **New Project**.
+![VoTT Home Screen](./media/object-detection-model-builder/vott.png)
+1. Update the following Project Settings:
+    1. Create a *Display Name* (like "StopSignObjDetection").
+    1. Select "Generate New Security Token" for *Security Token*.
+    1. Create a new Connection for the Source Connection by selecting **Add Connection**. Create a *Display Name* for the connection (like StopSignImages), and select "Local File System" as the Provider. For the *Folder Path*, select the folder which contains the 50 training images.
+    ![VoTT New Connection Dialog](./media/object-detection-model-builder/vott-new-connection.png)
```
Comment (Youssef1313) [inline docs/machine-learning/tutorials/object-detection-model-builder.md:74]: Surround image with blank lines.

File: docs/machine-learning/tutorials/object-detection-model-builder.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+## Choose a scenario
+
+1. In **Solution Explorer**, right-click the *StopSignDetection* project, and select **Add** > **Machine Learning**. This opens the Model Builder UI.
+1. For this sample, the scenario is object detection. In the *Scenario* step of Model Builder, select the **Object Detection** scenario.
+![Model Builder wizard in Visual Studio](./media/object-detection-model-builder/obj-det-scenario.png)
+
+    > If you don't see *Object Detection* in the list of scenarios, you may need to [update your version of Model Builder](https://linkhere.com).
+
+## Choose the training environment
```
Comment (Youssef1313) [inline docs/machine-learning/tutorials/object-detection-model-builder.md:120]:
```suggestion
## Choose the training environment

```

**Comment by BillWagner** [comment]
@briacht Are you going to return to this? Should it be closed?

**Comment by luisquintanilla** [comment]
@BillWagner let's keep this open for now. There's still a plan to publish, but there's some changes in the tooling that need to be updated in this tutorial. Thanks

**Pull request closed without merging**